.gitignore

```diff
@@ -10,16 +10,19 @@ fastlane/README.md
 fastlane/report.xml
 
 # Testing
-/test_results
-/coverage
+test_results/
+coverage/
+
+# YARD Documentation
```

Comment:
+1 for writing "YARD" with all upper case letters, because it's an acronym (Yay! Another Ruby Documentation tool) 🤓

Reply:
Yeah, I like [going the extra (0.00056818) mile](https://www.google.com/search?q=1+yard+in+miles) by using correct case 🥁 😆